test/Microsoft.Sbom.Api.Tests/SignValidator/SignValidationProviderTests.cs

```diff
@@ -31,15 +31,12 @@ public void SignValidationProvider_AddsValidator_Succeeds()
     }
 
     [TestMethod]
-    [ExpectedException(typeof(ArgumentNullException))]
     public void SignValidationProvider_NullValidators_Throws()
     {
         var mockOSUtils = new Mock<IOSUtils>();
         mockOSUtils.Setup(o => o.GetCurrentOSPlatform()).Returns(OSPlatform.Windows);
 
-        var signValidator = new SignValidationProvider(null, mockOSUtils.Object);
-        signValidator.Init();
-        signValidator.Get();
+        Assert.ThrowsException<ArgumentNullException>(() => new SignValidationProvider(null, mockOSUtils.Object));
```

Comment:
Is this code equivalent since it omits the Init and Get calls that the previous code made? Not sure where we were expecting the exception to occur previously.

Reply:
The exception is occurring in the constructor. It doesn't feel like that was the original intent so perhaps the original test was not actually testing what it was supposed to test

Reply:
Follow-up: The behavior being checked is indeed in the constructor. The calls to `Init` and `Get` were probably just an old copy/paste error